test/exe/main_common_test.cc

```diff
@@ -245,6 +267,7 @@ TEST_P(AdminRequestTest, AdminRequestGetStatsAndQuit) {
 TEST_P(AdminRequestTest, AdminRequestGetStatsAndKill) {
   startEnvoy();
   started_.WaitForNotification();
+  waitForEnvoyRun();
```

Comment:
Do we still need `started_` or can we get rid of that?

Reply:
It's not needed, but I want to leave it there to make it easy to undo by just deleting all of `waitForEnvoyRun()` later. I will leave a comment.